app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+
+### Non-eligibility for trainee teachers 
+
+You will not be eligible for the international relocation payment if you are: 
+
+* a UK citizen 
+
+* an Irish citizen 
+
+* a non-UK citizen who moved to England to live permanently more than 3 months before the start of your course 
+
+* doing an assessment only QTS 
+
+* training to teach only English language and literature
+
+## How trainee teachers can get the international relocation payment 
+
+### Fee-paying trainees 
+
+If you have a place on a teacher training course where you pay a fee, you do not need to apply for the international relocation payment (IRP). Your training provider will email you if your choice of course makes you eligible for the IRP. Your visa status and date of arrival will be checked by the Department for Education. If you are eligible, you’ll get the IRP around the end of your first term. 
+
+If you believe you meet the eligibility requirements, but have not been informed by your teacher training provider that you may be awarded the IRP, please contact your training provider directly first. If your training provider cannot help you, you can email us at teach.inengland@education.gov.uk. 
+
+### Salaried trainees 
+
+If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to let us know you are interested in applying for the IRP. You will be asked to fill in a form giving us your details and we’ll get in touch when IRP applications open for salaried trainees in autumn 2023. If you are eligible, you’ll get the IRP around the end of your first term. 
+
+## The international relocation payment for teachers  
+
+To be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.
+
+### 1. Teaching subject
+
+You must have accepted a job as a teacher of languages or physics in an English state secondary school on a contract lasting at least one academic year.  
+
+a) Teachers of all languages offered in English state secondary schools are eligible to apply for the IRP. The language or languages can be combined with another subject, but must make up at least 50% of teaching time. You can combine teaching English with another language, as long as the non-English language makes up at least 50% of your teaching timetable.  
+
+b) Physics can be taught with another subject, but must make up at least 50% of teaching time. Teachers of general science or combined science  are also eligible to apply for the IRP, as long as they are teaching the physics element of general science or combined science. General science or combined science can be taught with another subject, but must make up at least 50% of teaching time.  
+
+c) 'State schools' are schools funded by the UK government. 'Secondary schools' teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.  
```

Comment:
would this information be better served where you first mention state and secondary schools? This info could go in brackets after the 'You must have accepted a job ...' sentence

Reply:
I have restructured this section...